exec/java-exec/src/test/java/org/apache/drill/exec/fn/impl/TestCryptoFunctions.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.fn.impl;
+
+import org.apache.drill.BaseTestQuery;
+import org.junit.Test;
+
+public class TestCryptoFunctions extends BaseTestQuery {
+
+    @Test
+    public void testMD5() throws Exception {
+        final String query = "select md5( 'testing' ) as md5_hash from (values(1))";
+        testBuilder()
+            .sqlQuery(query)
+            .ordered()
+            .baselineColumns("md5_hash")
+            .baselineValues("ae2b1fca515949e5d54fb22b8ed95575")
+            .go();
+    }
+
+    @Test
+    public void testSHA1() throws Exception {
+        final String query = "select sha( 'testing' ) as sha_hash from (values(1))";
```

Comment:
Please remove extra space here and in test below: `sha( 'testing' ) ` -> `sha('testing') `